### PR TITLE
Denote commands with a /

### PR DESCRIPTION
**Pull request by dyc3**
This makes it so that commands must start with a `/`, and splits the input by spaces to get arguments for commands.

My reasoning for choosing `/` specifically is because users (especially gamers) will be more familiar with using `/` to denote commands (eg minecraft)

### CODE REVIEW

**Comment by nickwalton** [comment]
What happens if the input is the empty string. I believe you'll get an error.

**Comment by Renha** [comment]
It's not very convenient to type in / using say mobile phone

**Comment by spenserblack** [comment]
@Renha :thinking: This value should probably be something that nobody would *ever* start a sentence with, either purposefully or accidentally. I could see `.` being more convenient as a command escape \(the Node.js REPL uses this\) for both PC and mobile phone users, so maybe that should be considered.

But personally I agree with @dyc3 that it should be `/`. I have terrible typing accuracy on mobile, and an "inconvenient" character would definitely prevent me from making mistakes :laughing:

File: play.py [hunk 1]
```diff
@@ -145,82 +145,86 @@ def play_aidungeon_2():
...
-                    id = story_manager.story.save_to_storage()
-                    console_print("Game saved.")
-                    console_print(
-                        "To load the game, type 'load' and enter the following ID: "
-                        + id
-                    )
-                else:
-                    console_print("Saving has been turned off. Cannot save.")
+                    else:
+                        console_print(f"Invalid argument: {args[0]}")
```
Comment (spenserblack) [inline play.py]: Is the minimum Python version documented anywhere? f-strings require Python 3.6, which shouldn't conflict with the Colab runtime, but may conflict with users running AIDungeon locally.

Comment (dyc3) [reply play.py]: IIRC, the install script specifically checks for python 3.6

**Comment by dyc3** [comment]
@nickwalton Good catch, will fix.

@Renha We have mobile players? Even then, I'm pretty sure during a normal session commands wouldn't be used very often, so convenience shouldn't be a huge factor.

File: play.py (hunk 1 above)
Comment (spenserblack) [reply play.py]: Sorry, completely missed that. Thanks!

**Comment by ben-bay** [comment]
I'm good with this, after you add it to `CHANGELOG.md` :)

**Comment by dyc3** [comment]
@ben-bay done!

**Pull request merged**